### ISSUE TITLE
timeline of tasks disapeared

### ISSUE THREAD

**Issue by DukeHouse** [Leonezz/obsidian-tasks-calendar-wrapper#140]
<img width="358" alt="截屏2024-11-08 00 11 32" src="https://github.com/user-attachments/assets/90ab306a-e26b-4e5d-b1f2-a370ba1f515f">

**Comment by DukeHouse** [comment]
> <img alt="截屏2024-11-08 00 11 32" width="358" src="https://private-user-images.githubusercontent.com/52644701/384044191-90ab306a-e26b-4e5d-b1f2-a370ba1f515f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA5OTYyMDYsIm5iZiI6MTczMDk5NTkwNiwicGF0aCI6Ii81MjY0NDcwMS8zODQwNDQxOTEtOTBhYjMwNmEtZTI2Yi00ZTVkLWIxZjItYTM3MGJhMWY1MTVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA3VDE2MTE0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk5YzcwNGNmODE0NjNiMjBkZmMxYzBkOTM2MTlhODA0YjZmMGU5ZTQ0ZmFmMDBjZTFiZWJlYWFmYTM5MTdlZDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.JR2kgNIHcKvo5m9v1Pnp8SOrGqWaI4nif9zIMqqHLyg">

<img width="358" alt="截屏2024-11-08 00 12 18" src="https://github.com/user-attachments/assets/b29320f7-c0b4-4a50-8a2c-7f456c4f903a">

**Comment by DukeHouse** [comment]
obsidian version in v1.7.5 and obsidian-tasks-calendar-wrapper is latest

**Comment by MarkusRenner** [comment]
I face the same issue

**Comment by Leonezz** [comment]
Check if there are any wrong dates in your task items, it seems like to be the issue

**Comment by Djuris** [comment]
> Check if there are any wrong dates in your task items, it seems like to be the issue

What do you mean with wrong dates?  Does this mean i can`t have anything that looks like a date in the task apart from the date itself?

**Comment by Leonezz** [comment]
> > Check if there are any wrong dates in your task items, it seems like to be the issue
> 
> What do you mean with wrong dates? Does this mean i can`t have anything that looks like a date in the task apart from the date itself?

Yes you cannot have a ill formed date string as the value of any properties which are expected to be a valid date.
